modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ActionControls.js

```diff
@@ -84,6 +92,27 @@ const ActionControls = ({
 									symbol={item.icon}
 									title={item.label}
 								/>
+
+								{showDesignImprovements && (
+									<LinkOrButton
+										className="align-items-center d-lg-inline d-none mr-2 nav-link"
```

Comment:
I think we could reduce code duplication a bit, by having only one `LinkOrButton`, with its content conditionally changing. This is a minor thing, non-blocking.

Reply:
Thanks @markocikos! I started the task by doing this, but as required by other tasks in this epic, the `LinkOrButton` component already has a responsive mode where it displays either the icon or the text depending on the screen size, and adding a third mode that displays both simultaneously resulted in pretty ugly code. But I'll keep this in mind during the refactor of the MT that is in the roadmap!